website/content/en/docs/main/concepts/fabric.md

```diff
@@ -24,7 +24,7 @@ Currently, the concept of a Fabric is just that - a concept. It is not represent
 
 ### Prerequisites
 
-The following assume that you have access to the `clusterlink` CLI and one or more
+The following assume that you have access to the ClusterLink CLI and one or more
```

Comment:
the CLI command name is `clusterlink` and not `ClusterLink`. 
I don't think we should make this change